addok/core.py

```diff
@@ -205,9 +205,7 @@ def intersect(self, keys, limit=0):
             if len(keys) == 1:
                 ids = DB.zrevrange(keys[0], 0, limit - 1)
             else:
-                DB.zinterstore(self.pid, set(keys))
-                ids = DB.zrevrange(self.pid, 0, limit - 1)
-                DB.delete(self.pid)
+                ids = scripts.zinter(keys=set(keys), args=[self.pid, limit-1])
```

Comment:
Personally I'm not familiar with Redis call scripts. Not sure how to review this part.